workflow-templates/im-deploy-tf-auto-apply-main-to-dev-on-merge.yml

```diff
@@ -74,7 +74,34 @@ jobs:
         uses: im-open/setup-deploy-keys@v1.1
         with:
           deploy-key-info: ${{ env.SSH_DEPLOY_KEY_INFO }}
-          
+
+      - name: Install PS Modules
+        shell: pwsh
+        run: |
+          Install-Module -Name SqlServer -Force
+          Install-Module -Name Az -Force
+          Install-Module -Name Az.Accounts -MinimumVersion 2.2.0 -Force
+
+      - name: Confirm PS Modules
+        shell: pwsh
+        run: |
+          $modules = @("SqlServer", "Az", "Az.Accounts")
+          foreach ($module in $modules) {
+            if (Get-Module -ListAvailable -Name $module) {
+              Write-Host "$module Module exists"
+            }
+            else {
+              Write-Host "$module Module does not exist"
+            }
+          }
```

Comment:
I think only the SqlServer module needs to be installed at this point. The Az and Az.Accounts modules aren't used in the Terraform or anything else I see.

And won't the `Install PS Modules` step fail if it is unable to install? If not, we should try to make it do so and remove this `Confirm...` step. If we can't make the install step fail on failure, then at least this Confirm step should fail if it discovers something didn't get installed. As written, it will never fail.

Reply:
Sounds good to me.